invenio_rdm_records/models.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Bibliographic Record and Draft models."""
+
+from invenio_db import db
+from invenio_drafts_resources.drafts import DraftBase, DraftMetadataBase
+from invenio_records.api import Record
+from invenio_records.models import RecordMetadataBase
+
+
+class BibliographicRecordDraftMetadata(db.Model, DraftMetadataBase):
```

Comment:
I fully agree. As said above im honestly not convinced on ""Deposit" when referring to mixed draft + record concepts."

Since we have a bit of a messy naming, but we have a release. Shall we just merge as is naming wise. But open an issue and next tuesday we take this topic first thing?

Reply:
Taking as is for now and following up in: https://github.com/inveniosoftware/invenio-rdm-records/issues/142